### PR TITLE
Ray cats are now a subtype of cats

### PR DESCRIPTION
**Pull request by Erikafox**
## About The Pull Request
as title

## Why It's Good For The Game
I think it's more interesting to make this something rare rather than something all cats just. Do.

## Changelog

:cl:
balance: All cats are no longer radiation indicators - only a specific breed of them.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/cat.dm [hunk 1]
```diff
@@ -83,6 +76,68 @@
 	mob_size = MOB_SIZE_SMALL
 	collar_type = "kitten"
 
+/mob/living/simple_animal/pet/cat/raycat
+	name = "\improper raycat"
+	desc = "A member of a feline lineage lineage said to trace back to the Night Of Fire. A raycat is an adoring pet for the radiation adverse spacer."
```
Comment (goober3) [inline code/modules/mob/living/simple_animal/friendly/cat.dm]: > feline lineage lineage

laugh at this user

Comment (Erikafox) [reply code/modules/mob/living/simple_animal/friendly/cat.dm]: true....

**Comment by rye-rice** [comment]
i dont think most cats are going to be thrown into rads, so  you wont see it constantly, i think its just a nice thing to have around as a "huh, I didn't know that" sort of deal

**Comment by Apogee-dev** [comment]
yeah i agree with rye. it was also made a Universal Cat Thing as kind of a nod to both the Solarian penchant for genetic engineering and their history in the Night of Fire, which I found to be a nice touch.

**Comment by FalloutFalcon** [comment]
the whole point of the idea of real ray cats is that they become very common to detect radiation.

**Comment by FalloutFalcon** [comment]
gonna close this as like, not planned!

**Pull request closed without merging**